src/app/Instance/InstancesList/InstancesList.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FunctionComponent, useState } from "react";
+import {
+  Button,
+  Card,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from "@patternfly/react-core";
+import CreateInstance from "@app/Instance/CreateInstance/CreateInstance";
+import { Table, TableColumn } from "@app/components/Table";
+import { Instance } from "../../../types/Instance";
+import { Pagination } from "@app/components/Pagination/Pagination";
+import { useTranslation } from "react-i18next";
+
+interface InstancesListProps {
+  /** List of columns for the instances table */
+  columnNames: TableColumn[];
+  /** List of instances */
+  instances: Instance[];
+}
+
+export const InstancesList: FunctionComponent<InstancesListProps> = ({
+  columnNames,
+  instances,
+}) => {
+  const { t } = useTranslation(["openbridgeTempDictionary"]);
+
+  const actionResolver = () => {
+    return [
+      {
+        title: t("common.details"),
+        onClick: () =>
+          // @TODO missing action to perform when clicking on details action
+          {},
+      },
+      {
+        title: t("common.delete"),
+        onClick: () =>
+          // @TODO missing action to perform when clicking on delete action
+          {},
+      },
+    ];
+  };
+
+  const [showCreateInstance, setShowCreateInstance] = useState(false);
+
+  const getPagination = (
+    itemCount: number,
+    isCompact: boolean,
+    className: string
+  ) => (
+    <Pagination
+      className={className}
+      itemCount={itemCount}
+      page={1}
+      perPage={20}
+      isCompact={isCompact}
+      onChange={() =>
+        // @TODO missing action when changing the page
+        {}
+      }
+    />
+  );
+
+  return (
+    <Card>
+      <Toolbar>
+        <ToolbarContent>
+          <ToolbarItem alignment={{ default: "alignLeft" }}>
+            <Button onClick={() => setShowCreateInstance(true)}>
+              {t("instance.createSEInstance")}
+            </Button>
+            <CreateInstance
+              isLoading={false}
+              isModalOpen={showCreateInstance}
+              onClose={() => setShowCreateInstance(false)}
+              onCreate={() => setShowCreateInstance(false)}
+            />
+          </ToolbarItem>
+          <ToolbarItem
+            variant="pagination"
+            alignment={{ default: "alignRight" }}
+          >
+            {getPagination(
+              instances.length,
+              true,
+              "instances-list-page__pagination--top"
+            )}
+          </ToolbarItem>
+        </ToolbarContent>
+      </Toolbar>
+      <Table
+        actionResolver={actionResolver}
+        ariaLabel={t("openbridgeTempDictionary:instancesListTable")}
+        columns={columnNames}
+        cssClasses="tableInstances"
+        rows={instances}
+      />
+      {getPagination(
+        instances.length,
+        false,
+        "instances-list-page__pagination--bottom"
+      )}
```

Comment:
Thanks @vpellegrino for all the changes you already did.
Another minor observation from me. I only noticed now that the bottom pagination lacks some right padding, it looks too close to the right border of its container.
I believe you can solve this quickly without manually apply padding if you add `variant="bottom"` to the [PF Pagination component](https://www.patternfly.org/v4/components/pagination/#pagination) in this case.

Reply:
@kelvah thanks for your feedback. Added the `variant` as well.

Reply:
:tada: 

You can even get rid of the `instances-list-page__pagination--bottom` class now. All the padding should be already covered.